src/components/NewDesign/Modals/SelectModals/SelectTokenModal/SelectTokenModal.tsx

```diff
@@ -42,6 +41,13 @@ export const SelectTokenModal: React.FC<ISelectTokenModal> = ({
     setValue(e.target.value)
   }
 
+  const tokenIndex = (name: string) => {
+    return 'SOL BTC USD FTT ETH'.split(' ').findIndex((token) => {
```

Comment:
don't hardcode these

Reply:
Maybe we just fetch tokens array when we will have it on Redux? This solution solve problem with second array which we display fetch token pairs.